src/core/server.h

```diff
@@ -42,7 +42,6 @@
 #include "src/core/server_status.pb.h"
 #include "tensorflow/core/lib/core/status.h"
 #include "tensorflow_serving/model_servers/server_core.h"
-#include "tensorflow_serving/util/net_http/server/public/httpserver.h"
```

Comment:
In Dockerfile and tools/patch/tfs there are some patches we no longer need to apply because we don't use TFS http service anymore.